CONTRIBUTING.md

```diff
@@ -1,32 +0,0 @@
-# Contributing to list
```

Comment:
Let's keep this file and instead referencing it from the README, as it's quite standard to have a CONTRIBUTING.md